src/CancelRequestDialog.js

```diff
@@ -130,10 +140,14 @@ class CancelRequestDialog extends React.Component {
         />
         */}
         <TextArea
-          label={formatMessage(
-            { id: 'ui-requests.cancel.additionalInfoLabel' },
-            { required: reason.requiresAdditionalInformation ? '*' : ' ' }
-          )}
+          label={
+            <FormattedMessage
+              id="ui-requests.cancel.additionalInfoLabel"
+              values={{
+                required: reason.requiresAdditionalInformation ? '*' : ' '
+              }}
+            />
+          }
           placeholder={
```

Comment:
I think this pattern would work to be able to switch the `placeholder` away from `formatMessage`:

- In the render, before the `return`:
```
const additionalInfoPlaceholderKey = reason.requiresAdditionalInformation ?
      'ui-requests.cancel.additionalInfoPlaceholderRequired :
      'ui-requests.cancel.additionalInfoPlaceholderOptional';
```
- Then wrap the `TextArea`:
```
<FormattedMessage id={additionalInfoPlaceholderKey}>
  {placeholder => {
    <TextArea
      label={
        <FormattedMessage
          id="ui-requests.cancel.additionalInfoLabel"
          values={{
            required: reason.requiresAdditionalInformation ? '*' : ' '
          }}
        />
      }
      placeholder={placeholder}
      value={additionalInfo}
      onChange={this.onChangeAdditionalInfo}
    />
  )}
</FormattedMessage>
```

Reply:
Thanks, @cherewaty! I fixed the code in accordance with your advice